### PR TITLE
feat : Acces Previous year paper 

### PR DESCRIPTION
**Pull request by refactor-droidyy**

### CODE REVIEW

File: app/src/main/java/govind/iiitl/app/models/Uploads.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package govind.iiitl.app.models;
+
+public class Uploads {
+
+    public String name;
```
Comment (GOVINDDIXIT) [inline app/src/main/java/govind/iiitl/app/models/Uploads.java:5]: make private

Comment (refactor-droidyy) [reply app/src/main/java/govind/iiitl/app/models/Uploads.java:5]: @GOVINDDIXIT  I have checked this ... But still it's causing error and this has been resolved by using a pojo class . But the items are not still visible only their count can be seen

Comment (GOVINDDIXIT) [reply app/src/main/java/govind/iiitl/app/models/Uploads.java:5]: @refactor-droidyy Don't worry I will fix it 😃

**Comment by GOVINDDIXIT** [comment]
@refactor-droidyy pull latest changes and modify PR accordingly

**Pull request closed without merging**